harfbuzz-sys/Cargo.toml

```diff
@@ -1,17 +1,14 @@
 [package]
 name = "harfbuzz-sys"
-version = "0.6.0"
-edition = "2021"
-
-authors = ["The Servo Project Developers"]
-license = "MIT OR Apache-2.0"
 readme = "README.md"
-
-description = "Rust bindings to the HarfBuzz text shaping engine"
```

Comment:
All three crates' descriptions have been removed; I assume this is unintentional?
[Crates.io requires the description field be set.](https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field)